pkgs/os-specific/linux/nvidia-x11/settings.nix

```diff
@@ -68,6 +68,14 @@ stdenv.mkDerivation rec {
     ${lib.optionalString (!withGtk3) ''
       rm -f $out/lib/libnvidia-gtk3.so.*
     ''}
+
+    # Desktop file with icon.
+    sed -i doc/nvidia-settings.desktop \
+      -e "s|^Exec=.*$|Exec=nvidia-settings|" \
+      -e "s|^Icon=.*$|Icon=nvidia-settings|" \
```

Comment:
why do you need to replace the value ? is it using a full path ?

Reply:
The template has substitution variables that are supposed to be replaced resulting in absolute paths:
```
Exec=__UTILS_PATH__/nvidia-settings
Icon=__PIXMAP_PATH__/nvidia-settings.png
```
In NixOS we usually avoid absolute paths in desktop files because some desktops copy the desktop file when making favorites, and the path breaks on an upgrade.

Reply:
Add this to the commit message please and I will merge.